src/_zkapauthorizer/model.py

```diff
@@ -778,6 +779,32 @@ def get_latest_lease_maintenance_activity(self, cursor):
             parse_datetime(finished, delimiter=" "),
         )
 
+    @with_cursor
```

Comment:
It might be that the caller ends up being responsible for this transaction management because these inserts will need to run in the same transaction as the statements themselves come from.  I don't think there's a reason to change it now rather than later, just commenting to share my hunch about where this will have to go.

Reply:
okay. should i add a comment? (or we just remove the decorator when .. the other code appears?)